mscore/pathlistdialog.cpp

```diff
@@ -41,8 +42,12 @@ PathListDialog::PathListDialog(QWidget* parent)
 
 void PathListDialog::addClicked()
       {
-      QString newPath = QFileDialog::getExistingDirectory (this, tr("Choose a directory"),
-         QString("%1/%2").arg(QDesktopServices::storageLocation(QDesktopServices::DocumentsLocation)).arg(QCoreApplication::applicationName()));
+      QString newPath = QFileDialog::getExistingDirectory(
+         this,
+         tr("Choose a directory"),
+         QString("%1/%2").arg(QDesktopServices::storageLocation(QDesktopServices::DocumentsLocation)).arg(QCoreApplication::applicationName()),
+         preferences.nativeDialogs ? QFileDialog::Options() : QFileDialog::DontUseNativeDialog
```

Comment:
CORRECTION: Looks like I am able to compile with the full large statment, I believe because the QFileDialog::ShowDirsOnly acts as a cast.  I'll make that change for all and resubmit.